test/fuzz/build.sh

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash -eu
+#
+# SPDX-License-Identifier: BSD-3-Clause
+#
+#  Point Cloud Library (PCL) - www.pointclouds.org
+#  Copyright (c) 2020-, Open Perception
+#
+#  All rights reserved
+#
+
```

Comment:
It might make sense to state the assumptions made here:
* Script is run in a docker with pcl-build-deps present
* PCL is present in `WORKDIR/pcl`

Reply:
Sure, I will get that added!